keyboards/atreus/keymaps/quartz64/keymap.c

```diff
@@ -0,0 +1,147 @@
+*/
+
+  [_RS] = LAYOUT( /* RAISE  */
+    KC_PSCR, PWD1,           KC_UP,   PWD2,    KC_PGUP,                   KC_HOME, KC_F7,   KC_F8,  KC_F9,  _______,
+    _______, KC_LEFT,        KC_DOWN, KC_RGHT, KC_PGDN,                   KC_END,  KC_F4,   KC_F5,  KC_F6,  _______,
+    _______, MC2,            MC3,     KC_LPRN, KC_RPRN,                   KC_RALT, KC_F1,   KC_F2,  KC_F3,  _______,
+    _______, LCTL(KC_LSFT),  _______, MC1,     _______, _______, _______, _______, KC_F10,  KC_F11, KC_F12, _______
+    ),
+
+/*
+	,----------------------------------.              ,----------------------------------.
+	|   !  |   @  |   #  |   $  |  %   |              |   /  |   7  |   8  |   9  |  .   |
+	|------+------+------+------+------|              |------+------+------+------+------|
+	| trns |   ^  |   &  |   *  | PWD3 |              |   *  |   4  |   5  |   6  | trns |
+	|------+------+------+------+------|------.,------|------+------+------+------+------|
+	| trns |r_tog | r_hue| r_sat| r_val|      ||      |   +  |   1  |   2  |   3  | trns |
+	|------+------+------+------+------|      ||      |------+------+------+------+------|
+	| trns | Esc  |RESET |capslk| trns |------'`------| trns |   0  |   -  |  +=  | trns |
+	`----------------------------------'              `----------------------------------'
+*/
+
+  [_LW] = LAYOUT( /* LOWER */
+    KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC,                   KC_PSLS, KC_7, KC_8,    KC_9,   KC_PDOT,
+    _______, KC_CIRC, KC_AMPR, KC_ASTR, PWD3,                      KC_PAST, KC_4, KC_5,    KC_6,   _______,
+    _______, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI,                   KC_PPLS, KC_1, KC_2,    KC_3,   _______,
+    _______, KC_ESC , RESET,   KC_CAPS, _______, _______, _______, _______, KC_0, KC_PMNS, KC_EQL, _______
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+	  case MC1:
+      // French quotes
+      if (record->event.pressed) {
+        SEND_STRING(SS_TAP(X_RALT)"<<"SS_TAP(X_RALT)">>"SS_TAP(X_LEFT));
+      }
+      return false;
+      break;
+	  
+	  case MC2:
+      // NB-space + mdash + space
+      if (record->event.pressed) {
+        SEND_STRING(SS_TAP(X_RALT)"  "SS_TAP(X_RALT)"--- ");
+      }
+      return false;
+      break;
+	  
+	  case MC3:                                                                                                             
+      // NB-space
+      if (record->event.pressed) {
+        SEND_STRING(SS_TAP(X_RALT)"  ");
+      }
+      return false;
+      break;
+	  
+	  case PWD1:
+      if (record->event.pressed) {
+        SEND_STRING("NakedLunch1991"SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
+	  
+	  case PWD2:
+      if (record->event.pressed) {
+        SEND_STRING("O94nx4sUWHc4akud"SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
+	  
+	  case PWD3:
+      if (record->event.pressed) {
+        SEND_STRING("Q123qQ123q"SS_TAP(X_ENTER));
```

Comment:
You can do this and save some bytes:
```suggestion
        SEND_STRING("Q123qQ123q\n");
```